client/src/components/Nav/SettingsTabs/General.tsx

```diff
@@ -85,6 +85,7 @@ export const LangSelector = ({
   const languageOptions = [
     { value: 'auto', display: localize('com_nav_lang_auto') },
     { value: 'en-US', display: localize('com_nav_lang_english') },
+    { value: 'tr-TR', display: localize('com_nav_lang_turkish') },
```

Comment:
can you move it under the vietnamese one?

Reply:
Yes, i moved it.